html-syntax/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Syntax</title>
+</head>
+<body>
+  <img alt = "Greninja" src = "https://cdn.bulbagarden.net/upload/thumb/6/67/658Greninja.png/1200px-658Greninja.png">
```

Comment:
There should be no spaces around the equal sign in attributes

Reply:
Okay, I'll change that